app/scripts/lib/transaction/eip5792.ts

```diff
@@ -31,7 +31,7 @@ const VERSION_GET_CALLS_STATUS = '1.0';
 export async function processSendCalls(
   hooks: {
     addTransactionBatch: TransactionController['addTransactionBatch'];
-    getDisabledAccountUpgradeChains: () => Hex[];
+    getDisabledAccountUpgradeChainsAddresses: () => Record<Hex, Hex[]>;
```

Comment:
Minor, maybe `getDisabledUpgradeAccountsByChain`?

Reply:
PR updated to address this